### PR TITLE
Runner docs

### PR DESCRIPTION
**Pull request by thomasr8**
Added Runner pages to side navigation

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Comment by teesloane** [comment]
This is blocked by #4651 #4660 and #4645 I believe.

**Comment by rosieyohannan** [comment]
> This is blocked by #4651 #4660 and #4645 I believe.

yep :-) just working to get everything in one branch now...

File: jekyll/_data/sidenav.yml [hunk 1]
```diff
@@ -154,6 +154,15 @@ en:
           link: 2.0/private-images/
         - name: Xcode Image Policy
           link: 2.0/xcode-policy/
+    - name: Runner
+      link: 2.0/runner-overview/
+      children:
```
Comment (rosieyohannan) [inline jekyll/_data/sidenav.yml:159]:
```suggestion
      children:
        - name: Runner Overview
           link: 2.0/runner-overview/
```

**Comment by rosieyohannan** [comment]
In fact, I think I'll merge all the docs into this branch

File: jekyll/_cci2/runner-installation.adoc [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+```
+[Unit]
+Description=CircleCI Runner
+After=network.target
+[Service]
+ExecStart=/opt/circleci/circleci-launch-agent --config /opt/circleci/launch-agent-config.yaml
+Restart=always
+User=root
+NotifyAccess=exec
+TimeoutStopSec=600
```
Comment (glenjamin) [inline jekyll/_cci2/runner-installation.adoc:252]: Could we change this to 18300 please?

for https://circleci.atlassian.net/browse/CIRCLE-31045

File: jekyll/_cci2/runner-installation.adoc [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+systemd-run \
+    --pipe --collect --quiet --wait \
+    --uid "$USER_ID" --unit "$unit" -- "$@"
+```
+
+=== Enable the `systemd` unit
+
+Create `/opt/circleci/circleci.service` owned by `root` with permissions `755`.
+
+You must ensure that `TimeoutStopSec` is greater than the total amount of time the `stop-agent` script will take.
```
Comment (glenjamin) [inline jekyll/_cci2/runner-installation.adoc:239]:
```suggestion
You must ensure that `TimeoutStopSec` is greater than the total amount of time a task will run for - which defaults to 5 hours.
```

**Review by glenjamin — APPROVED** [review]
Will try and do another full pass, but here's a little update for CIRCLE-31045

File: jekyll/_cci2/runner-installation.adoc [hunk 4]
```diff
@@ -0,0 +1,467 @@
...
+```
+[Unit]
+Description=CircleCI Runner
+After=network.target
+[Service]
+ExecStart=/opt/circleci/circleci-launch-agent --config /opt/circleci/launch-agent-config.yaml
+Restart=always
+User=root
+NotifyAccess=exec
+TimeoutStopSec=600
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation.adoc:254]:
```suggestion
TimeoutStopSec=18300
```

**Pull request merged**